Actions/AL-Go-Helper.ps1

```diff
@@ -226,6 +226,28 @@ function Expand-7zipArchive {
     }
 }
 
+#
+# Download and import the BcContainerHelper module
+#
+# ContainerHelperVersion can be:
+# - preview (or dev), which will use the preview version downloaded from bccontainerhelper blob storage
+# - latest, which will use the latest version downloaded from bccontainerhelper blob storage
+# - a specific version, which will use the specific version downloaded from bccontainerhelper blob storage
+# - none, which will use the BcContainerHelper module installed on the build agent
+# - private, which will use a fork of BcContainerHelper from the same owner as the AL-Go repo
```

Comment:
Should we remove this if 'private' is no longer supported?

Reply:
Yes